client/src/components/nodesPanel.js

```diff
@@ -91,13 +91,33 @@ export default class NodesPanel extends Base {
     }
 }
 
+/**
+ * 
+ * @param {*status* object with a status field  (most likely the row from the TableBody)}
+ * @returns the status text, as defined in https://kubernetes.io/docs/concepts/architecture/nodes/#condition
+ */
 function getReadyStatus({status}) {
     if (!status.conditions) return null;
-
     const ready = status.conditions.find(y => y.type === 'Ready');
     return ready && ready.status;
 }
 
+/** Simple mapping between ready statuses and an UTF-8 symbol character */
+const statusesToUtf8 = { "True": "\u2713", "False": "\uD83D\uDEC7", "Unknown": "\u003F" }
+
+/**
+ * 
+ * @param {*statusTxt a status text (as returned by getReadyStatus for example) }
+ * @returns a dedicated span element with an UTF-8 symbol representing the status
+ */
+function statusSymbol(statusTxt) {
```

Comment:
I wonder if it would be worth pulling this out into a dedicated component so it could be used elsewhere if needed later? Not critical for sure.

Reply:
Yes, I was wondering whether the "statusTxt" was specific to the kubernetes "Node" (that's why I've created dedicated css classes with the name "node" in it). But if you think we could reuse this elsewhere, I'll extract it in a dedicated component ;)

Reply:
Nah, let's leave it as is for now. We can always extract it later if/when the need arrises.